### DISCUSSION TITLE
Correct way to link a cask to github actions

### DISCUSSION THREAD

**Discussion by spookyuser** [Homebrew/discussions#3795] [Writing Formulae/Casks] [answered]
Hi! 

I noticed that the cask for [treesheets](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/treesheets.rb) doesn't work on m1 macs, and I think it's because it's linked to an old version, whereas newer versions are being published to it's github release page from github actions over [here](https://github.com/aardappel/treesheets/releases)

So just wondering what the best way of linking to the releases on the github page is :) do I set the url to the most recent version always or should the version be bumped manually

**Comment by SMillerDev** [comment, marked as answer]
There is no way to link them, but you can run `brew bump` or `brew bump-cask-pr` to update a cask.
